test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -108,9 +108,12 @@ const PossibleIssues = () => {
         if (buildName.includes('j9') || buildName.includes('ibm')) {
             additionalRepo = '+repo:eclipse-openj9/openj9';
         }
+        else if (buildId.includes('hs')){
```

Comment:
Could you let me know if you tested this change? `buildId`  is an ID. It does not contain `hs`.